src/lapis/mime.types

```diff
@@ -0,0 +1,81 @@
+types {
```

Comment:
Don't really know what this file is for. The `lapis new` site generator included it.